ex2/ex2.c

```diff
@@ -8,7 +8,23 @@
 
 int main(int argc, char* argv[])
 {
-    // Your code here 
-    
+    FILE *fp;
+    int c;
+
+    fp = fopen("text.txt", "r");
```

Comment:
Opening the text file with the 'r' flag won't allow you to write to it. It looks like you printed out what was stored in the text file instead of writing to the file.